README.md

```diff
@@ -0,0 +1,32 @@
+# de.ifw_dresden.nagstamon
```

Comment:
I'm not a fan of such READMEs as they're pretty much duplicating existing documentation (which should be improved instead if it's not concise enough for quickstart)